mocks/helpers/buildQuerySnapShot.d.ts

```diff
@@ -0,0 +1,18 @@
+import type { MockedDocument, DocumentHash } from './buildDocFromHash';
+
+export interface MockedQuerySnapshot<Doc = MockedDocument> {
+  empty: boolean;
+  size: number;
+  docs: Array<Doc>;
+  forEach(callbackfn: (value: Doc, index: number, array: Doc[]) => void): void;
+  docChanges(): {
```

Comment:
This is about to be updated in https://github.com/Upstatement/firestore-jest-mock/pull/91.  As @CorentinDoue pointed out, this should be returning an array, rather than an array-like object

Reply:
Fixed.